app/views/categories/_form.html.erb

```diff
@@ -0,0 +1,23 @@
+<%= form_with(model: category) do |form| %>
+  <% if category.errors.any? %>
+    <div style="color: red">
+      <h2><%= pluralize(category.errors.count, "error") %> prohibited this post from being saved:</h2>
+
+      <ul>
+        <% category.errors.each do |error| %>
+          <li><%= error.full_message %></li>
+        <% end %>
+      </ul>
+    </div>
+  <% end %>
+
+  <div>
+    <%= form.text_field :name, placeholder: "Name" %>
+  </div>
+  <div>
+    <%= form.file_field :icon %>
+  </div>
+  <div class="category-submit">
+    <%= form.submit %>
```

Comment:
- [Optional] Please kindly fix this button too. It should be in line with the requirements.

 
![Screenshot from 2023-04-29 15-21-04](https://user-images.githubusercontent.com/63087491/235307791-5bf04192-c122-4667-9ac4-82005035da70.png)